websub-native/src/main/java/org/ballerinalang/net/websub/BallerinaWebSubConnectorListener.java

```diff
@@ -375,9 +372,9 @@ private BString getParamStringValue(BMap<BString, Object> params, BString key) {
             return StringUtils.fromString("");
```

Comment:
If it is constant value we are passing to fromString method, we can replace them with BString constant so we do not need to create Bstring everytime.